### PR TITLE
Add: Galar Newcomers Online Competition

### PR DESCRIPTION
**Pull request by cantsay**

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -281,6 +281,37 @@ let Formats = [
 				if (!allowedGigantamaxes.includes(pokemon.species)) pokemon.canGigantamax = null;
 			}
 		},
+	},
```
Comment (KrisXV) [inline config/formats.js]:
```suggestion
```

File: config/formats.js [hunk 2]
```diff
@@ -281,6 +281,37 @@ let Formats = [
...
+		{
```
Comment (LegoFigure11) [inline config/formats.js]:
```suggestion
	{
```
```
 285:1   error  Expected indentation of 1 tab but found 2      indent
```

File: config/formats.js [hunk 3]
```diff
@@ -281,6 +281,37 @@ let Formats = [
...
+				'Wooloo', 'Dubwool', 'Chewtle', 'Drednaw', 'Yamper', 'Boltund', 'Rolycoly', 'Carkol', 'Coalossal', 'Applin', 'Flapple', 'Appletun',
+				'Silicobra', 'Sandaconda', 'Cramorant', 'Arrokuda', 'Barraskewda', 'Toxel', 'Toxtricity', 'Sizzlipede', 'Centiskorch', 'Clobbopus', 'Grapploct', 'Sinistea', 'Polteageist',
+				'Hatenna', 'Hattrem', 'Hatterene', 'Impidimp', 'Morgrem', 'Grimmsnarl', 'Obstagoon', 'Meowth-Galar', 'Perrserker', 'Zigzagoon-Galar', 'Linoone-Galar', 'Obstagoon', 'Corsola-Galar',
+				'Cursola', 'Farfetch\'d-Galar', 'Sirfetch\'d', 'Mr. Mime-Galar', 'Mr. Rime', 'Yamask-Galar', 'Runerigus', 'Milcery', 'Alcremie', 'Falinks',
+				'Pincurchin', 'Snom', 'Frosmoth', 'Stonjourner', 'Eiscue', 'Indeedee', 'Morpeko', 'Cufant', 'Copperajah', 'Dracozolt', 'Arctozolt', 'Dracovish',
+				'Arctovish', 'Duraludon', 'Dreepy', 'Drakloak', 'Dragapult',
+			];
+			let template = Dex.getTemplate(set.species);
+			if (!newMonsList.includes(template.species)) return [`${template.species} is banned.`];
+		},
```
Comment (KrisXV) [inline config/formats.js]: there is a much better way to do this:

```javascript
onValidateSet(set) {
	let template = Dex.getTemplate(set.species);
	if (template.gen < 8) {
		return [`Only Pokemon from Generation 8 are allowed.`, `(${set.species} is from Generation ${template.gen}.)`];
	}
},
```

File: config/formats.js [hunk 4]
```diff
@@ -282,6 +282,30 @@ let Formats = [
...
+
+		mod: 'gen8',
+		gameType: 'doubles',
+		forcedLevel: 50,
+		teamLength: {
+			validate: [4, 6],
+			battle: 4,
+		},
+		ruleset: ['Standard GBU'],
+		banlist: ['Zacian', 'Zamazenta', 'Eternatus'],
```
Comment (LegoFigure11) [inline config/formats.js]:
```suggestion
```
```
Error: gen8galarnewcomers: Rule "-Zacian" was added by "[Gen 8] Galar Newcomers" but already exists in "Standard GBU"
```

File: config/formats.js [hunk 5]
```diff
@@ -282,6 +282,30 @@ let Formats = [
...
+		unbanlist: [
+			'Corviknight-Gmax', 'Orbeetle-Gmax', 'Drednaw-Gmax', 'Coalossal-Gmax', 'Flapple-Gmax', 'Appletun-Gmax', 'Sandaconda-Gmax',
+			'Centiskorch-Gmax', 'Hatterene-Gmax', 'Grimmsnarl-Gmax', 'Alcremie-Gmax', 'Copperajah-Gmax', 'Duraludon-Gmax',
+		],
+		onValidateSet(set) {
+			let template = Dex.getTemplate(set.species);
+			if (template.gen < 8) {
```
Comment (KrisXV) [inline config/formats.js]:
```suggestion
			if (template.gen < 8 || (template.isGigantamax && Dex.getTemplate(template.baseSpecies).gen < 8)) {
```

**Pull request merged**